components/hypercert/hypercert-minting-form/index.tsx

```diff
@@ -202,37 +202,24 @@ export function HypercertMintingForm({
         blueprint_minter_address: true,
       });
 
-  const defaultValues = presetValues
-    ? {
-        ...presetValues,
-        allowlistEntries: presetValues.allowlistEntries?.map((entry) => ({
-          ...entry,
-          units: BigInt(entry.units),
-        })),
-        projectDates: {
-          to: new Date(presetValues.projectDates.to),
-          from: new Date(presetValues.projectDates.from),
-        },
-      }
-    : value;
-
   const form = useForm<HypercertFormValues>({
     resolver: zodResolver(formSchemaUsed),
-    defaultValues,
+    defaultValues: value,
```

Comment:
The singular <> plural is irritating me a bit. Can we rename the local storage variable to plural?

Reply:
ne